recipes-devtools/rust/compiler-rt_1.10.0.bb

```diff
@@ -26,5 +32,12 @@ do_compile () {
 
 do_install () {
 	mkdir -p ${D}${libdir}
+}
+
+do_install () {
```

Comment:
Why is `do_install()` duplicated? I think this was probably meant to be squished together?